testing/python/language/test_tilelang_language_pdl.py

```
@@ -54,5 +55,50 @@ def test_pdl_sync():
     assert "__restrict__" not in code
 
 
+def _lower_cutedsl_for_pdl(program):
+    try:
+        from cutlass.cute import arch as cute_arch
+        from tilelang.jit.adapter.cutedsl.checks import check_cutedsl_available
+        from tilelang.jit.adapter.cutedsl.wrapper import TLCuTeDSLSourceWrapper
+        from tilelang.utils.target import determine_target
+
+        check_cutedsl_available()
+    except Exception as err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)